config/initializers/omniauth.rb

```diff
@@ -12,7 +12,7 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
🚲 

Honestly this is why rubocop is both great and terrible. I don't want parentheses in
```unless foo && bar
        do_thing
     end
```

But I DO want them in 
```
do_thing unless foo && bar
```

because you can more easily get into precedence issues with the latter. I don't think rubocop treats them differently, so it might be better to just drop the cop and watch for the former manually.